mesh_ops/MinimalHoleFill.cs

```diff
@@ -119,11 +121,15 @@ public bool Apply()
             int zero_collapse_passes = 0;
             int collapse_passes = 0;
             while (collapse_passes++ < 20 && zero_collapse_passes < 2) {
+                if (progressCancel != null && progressCancel.Cancelled())
+                    return false;
```

Comment:
why dont we put such condition into other While loops below?

Reply:
They aren't expensive to the same extent. Added